test/components/CalendarDay_spec.jsx

```diff
@@ -44,6 +44,10 @@ describe('CalendarDay', () => {
       handleDayClickSpy = sinon.spy(CalendarDay.prototype, 'handleDayClick');
     });
 
+    afterEach(() => {
```

Comment:
@goatslacker It gets included in the Webpack build as far as I can tell, but I still got double-wrap errors from sinon that necessitated this change.